server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java

```diff
@@ -438,6 +438,7 @@ private List<Database> getDatabaseByIds(Set<Long> ids) {
         Map<Long, ConnectionConfig> id2Connection =
                 dataSourceService.internalListSkipUserCheck(connectionIds, false, false)
```

Comment:
fullFillAttributes method is executed in dataSourceService.internalListSkipUserCheck

Reply:
may fix the same bug.